neurom/check/neuron_checks.py

```diff
@@ -287,3 +287,27 @@ def is_dangling(neurite):
     bad_ids = [(neurite.root_node.id, [neurite.root_node.points[1]])
                for neurite in iter_neurites(n) if is_dangling(neurite)]
     return CheckResult(len(bad_ids) == 0, bad_ids)
+
+
+def has_no_narrow_dendritic_section(neuron, radius_threshold=0.05,
```

Comment:
can we have instead has_no_narrow_neurite_section ?
with a parameter than can select which neurite type are taken into account ?

Reply:
Good point. Added